### ISSUE TITLE
`ClimaCoupler-AMIP` pipeline crashing with `saturation adjustment error`

### ISSUE THREAD

**Issue by sriharshakandala** [CliMA/ClimaCoupler.jl#1102]
The `ClimaCoupler-AMIP` pipeline crashing with a `saturation adjustment error` with the message `maxiter reached in saturation_adjustment_phq:`. 
Please see the build https://buildkite.com/clima/climacoupler-amip/builds/102#_ for details.

cc: @juliasloan25 @szy21

**Comment by szy21** [comment]
Yes, this simulation is unstable. We are not working on fixing it now but will look into it later.
